modules/liburing/2.5/patches/add_build_file.patch

```diff
@@ -0,0 +1,37 @@
+--- /dev/null
++++ a/BUILD.bazel
+@@ -0,0 +1,26 @@
++_COPTS = [
++    "-D_GNU_SOURCE",
++    "-D_LARGEFILE_SOURCE",
++    "-D_FILE_OFFSET_BITS=64",
++    "-DLIBURING_INTERNAL",
++    "-O3",
```

Comment:
Shouldn't this be left to `-c opt`?

Reply:
Given the nature of this library, it is always on the hot path and performance always critical. That is similar to packages like pigz where that optimization is specified as well.

Are there guidelines to follow that I can lookup to decide when this is or isn't appropriate to use?

Reply:
We don't have concrete guidelines yet. In this case this looks fine, thanks for explaining the reason.